.github/workflows/test-macos.yml

```diff
@@ -0,0 +1,20 @@
+name: CI Build MacOS
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
You have temporary tried to use `*/5` instead of just `*` - this looks like what I remembered how we once solved this.
Did this not work?
At least making the last segment a `*` instead of `0` is correct.
Now, we run every day at midnight.

Reply:
We can use `0 */5 * * *` but this would make the build run every 5 hours. 
https://crontab.guru/#0_*/5_*_*_*

We could also use `0 0-4 * * *`, but I don't know if Github puts the runs into a queue when the server load is high. Because with this cron schedule we would have 4 builts in a row.
https://crontab.guru/#0_0-4_*_*_*

Finally I created [check-commit.yml ](https://github.com/alfeilex/ide/blob/nightly-workflows/.github/workflows/check-commit.yml) which checks when the last commit was. Thus, with `use` there is no code duplication in the other workflows.